images/linux/scripts/installers/graalvm.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+source $HELPER_SCRIPTS/install.sh
+
+#Install GraalVM
+tmp_graalvm=/tmp/graalvm
```

Comment:
Why do we need the temp directory?
You can just pass empty string to `download_with_retries` and it will download directly to `/tmp`

Reply:
Fixed it